src/webui/www/private/download.html

```diff
@@ -117,15 +117,17 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
                             <label for="dlLimit">QBT_TR(Limit download rate)QBT_TR[CONTEXT=HttpServer]</label>
```

Comment:
I think you need to change this too
```html
<label for="dlLimitText">
```
and at other similar places.

Reply:
Sorry I forgot it